components/event-sources/reconciler/object/service_test.go

```diff
@@ -1,131 +1,85 @@
-/*
```

Comment:
Is the removal of license intentional? Why is that?

Reply:
See: https://github.com/kyma-project/kyma/pull/9186#discussion_r468424631